doc/source/myrocks/zenfs.rst

```diff
@@ -243,23 +243,97 @@ The following command backs up everything from the root of the ZenFS drive:
 
 .. sourcecode:: bash
 
-    zenfs backup --zbd=${NULLB} --path="/home/user/bkp" --backup_path=./
+    $ zenfs backup --zbd=${NULLB} --path="/home/user/bkp" --backup_path=./
```

Comment:
Let's normalize backup_path / restore_path in examples - `/` is a more straightforward value meaning "root"
```suggestion
    $ zenfs backup --zbd=${NULLB} --path="/home/user/bkp" --backup_path=/
```

Reply:
Done.